addons/skin.estuary/xml/Variables.xml

```diff
@@ -744,13 +755,78 @@
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,mpeg1)">MPEG-1</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,mpeg2)">MPEG-2</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,mpeg2video)">MPEG-2</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,mp4v)">MPEG-4</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,mpeg4)">MPEG-4</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,theora)">Theora</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,vc1)">VC-1</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,vc-1)">VC-1</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,vp8)">VP8</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,vp9)">VP9</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,wmv)">WMV</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,wmv3)">WMV</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,wvc1)">VC-1</value>
+		<value condition="String.IsEqual(VideoPlayer.VideoCodec,xvid)">XviD</value>
+		<value>$INFO[VideoPlayer.VideoCodec]</value>
+	</variable>
+	<variable name="VideoListHDRVar">
+		<value condition="String.IsEqual(ListItem.HdrType,dolbyvision)">DoVi HDR</value>
 		<value condition="String.IsEqual(ListItem.HdrType,hdr10)">HDR10</value>
 		<value condition="String.IsEqual(ListItem.HdrType,hlg)">HLG</value>
 		<value>$INFO[ListItem.HdrType]</value>
 	</variable>
-	<variable name="AudioCodecVar">
+	<variable name="VideoPlayerHDRVar">
+		<value condition="String.IsEqual(VideoPlayer.HdrType,dolbyvision)">DoVi HDR</value>
+		<value condition="String.IsEqual(VideoPlayer.HdrType,hdr10)">HDR10</value>
+		<value condition="String.IsEqual(VideoPlayer.HdrType,hlg)">HLG</value>
+		<value>$INFO[ListItem.HdrType]</value>
+	</variable>
+	<variable name="VideoListAspectVar">
+		<value condition="String.IsEqual(ListItem.VideoAspect,1.00)">1.00:1</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,1.19)">1.19:1</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,1.33)">4:3</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,1.37)">1.37:1</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,1.66)">1.66:1</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,1.78)">16:9</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,1.85)">1.85:1</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,2.00)">2.00:1</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,2.20)">2.20:1</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,2.35)">2.35:1</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,2.40)">2.40:1</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,2.55)">2.55:1</value>
+		<value condition="String.IsEqual(ListItem.VideoAspect,2.76)">2.76:1</value>
+		<value>$INFO[ListItem.VideoAspect,,:1]</value>
+	</variable>
```

Comment:
Couldn't this be simplified?
```
	<variable name="VideoListAspectVar">
		<value condition="String.IsEqual(ListItem.VideoAspect,1.33)">4:3</value>
		<value condition="String.IsEqual(ListItem.VideoAspect,1.78)">16:9</value>
		<value>$INFO[ListItem.VideoAspect,,:1]</value>
	</variable>
```

Seems a little inconsistent to add  :1 for all besides these 2 but I assume it's just due to what people expect to see regarding 4 by 3 and 16 by 9

Same with below:

```
	<variable name="VideoPlayerAspectVar">
		<value condition="String.IsEqual(VideoPlayer.VideoAspect,1.33)">4:3</value>
		<value condition="String.IsEqual(VideoPlayer.VideoAspect,1.78)">16:9</value>
		<value>$INFO[ListItem.VideoAspect,,:1]</value>
	</variable>
```

Reply:
I went with what I thought would be more user friendly for those, instead of keeping it consistant in this case. I have a lot of old british shows on DVD and they all have 4:3 on the back, never personally seen 1.33 used anywhere. For 1.78 I do have DVD's with that on the back, but by far the majority use 16:9.

I don't have any hard conviction in doing these aspects this way, so if others prefer to keep it consistant I'm ok with that. At least it being in a xml now makes it editable.

Reply:
Oh and yes it could be simplified exactly as you show. I did have it exactly like that in a early version, however one of my aims was to make everything user editable, hence going with the full list to make it easier for any user wishing to do this.

Reply:
I think consistency is important here. 

If you're going for screen size (16:9) over film aspect (1.78) as being more user friendly, then you should convert aspects to screen sizes for all common sizes not just 4:3 and 16:9.

Film aspect has the advantage of common denominator to compare easily -- e.g. I know from 2.33 vs 1.78 which is wider, but not necessarily between 2.33:1 and 16:9.

It's difficult because I think film aspect is more technically correct *for things shot on film*, whereas screen size would be more correct for things shot on video (e.g. for TV -- which is why you also see it on DVD and VHS where film has been converted to video and cropped for a smaller format).

Reply:
Ok that's me persuaded to keep consistency. I will revert back to an earlier simpler version of code that I had for the Aspect Ratio flags that didn't use the variables at all.

Reply:
Updated to the simpler method I have in a earlier version of this work.